src/entries/popup/pages/home/Points/utils.ts

```diff
@@ -0,0 +1,106 @@
+import { i18n } from '~/core/languages';
+
+import {
+  CATEGORY_DISPLAY_TYPE,
+  CATEGORY_TYPE,
+  EXISTING_USER_ERROR,
+  INVALID_REFERRAL_CODE_ERROR,
+} from './references';
+
+export const CHARACTER_TYPING_SPEED = 0.1;
```

Comment:
IMO this is a bit too slow. I'd lower it down to 0.05 or check what the app is using cause it def. seems faster there.

Reply:
i followed the app that had 0.1

Reply:
oh i just found 20 in other place, switching